go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/stretchr/testify v1.8.4
 	golang.org/x/sync v0.3.0
-	google.golang.org/protobuf v1.31.1-0.20230727123859-6d0a5dbd9500
+	google.golang.org/protobuf v1.31.1-0.20231027082548-f4a6c1f6e5c1
```

Comment:
This latest version of protobuf-go pulls in the latest `descriptor.proto` changes.